docs/en/ingest-management/release-notes/release-notes-8.14.asciidoc

```diff
@@ -0,0 +1,248 @@
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.14.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.14.0 relnotes
+
+[[release-notes-8.14.0]]
+== {fleet} and {agent} 8.14.0
+
+Review important information about {fleet-server} and {agent} for the 8.14.0 release.
+
+[discrete]
+[[security-updates-8.14.0]]
+=== Security updates
+
+{fleet-server}::
+* Update {fleet-server} Go version to 1.21.10. {fleet-server-pull}3528[#3528]
+
+{agent}::
+* Update {agent} Go version to 1.21.10. {agent-pull}4718[#4718]
+* Update all `opentelemetry-collector-contrib` packages. {agent-pull}4572[#4572]
+
+[discrete]
+[[new-features-8.14.0]]
+=== New features
+
+The 8.14.0 release added the following new and notable features.
+
+{fleet}::
+* Adds subfeatures privileges for {fleet}, {agents}, agent policies, and settings, this feature is in technical preview. ({kibana-pull}179889[#179889]).
+* Implements state machine behavior for package install. ({kibana-pull}178657[#178657])
+* Lowers the default `total_fields` limit to 1000 from 10k. ({kibana-pull}178398[#178398])
+* Avoids subobject and scalar mapping conflicts by setting `subobjects: false` on custom integrations. ({kibana-pull}178397[#178397])
+* Adds functionality to `default_fields` field, so a query can run against all fields in the mapping. ({kibana-pull}178020[#178020])
+* Relaxes delete restrictions for managed content installed by {fleet}. ({kibana-pull}179113[#179113])
+
+{agent}::
+* The Kubernetes secrets provider has been improved to update a Kubernetes secret  when the secret value changes. {agent-pull}4371[#4371] {agent-issue}4168[#4168]
+* The OpenTelemetry link:https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/filterprocessor[filterprocessor] is now available to users running {agent} in `otel` mode. {agent-pull}4708[#4708]
+* The OpenTelemetry link:https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/elasticsearchexporter[elasticsearchexporter] is now available to users running {agent} in `otel` mode. {agent-pull}4707[#4707]
+
+[discrete]
+[[enhancements-8.14.0]]
+=== Enhancements
+
+{fleet}::
+* Adds support for dimension mappings in dynamic templates. ({kibana-pull}180023[#180023])
+* Adds CPU metrics to request diagnostics. ({kibana-pull}179819[#179819])
+* Adds Settings Framework API and UI. ({kibana-pull}179795[#179795])
+* Adds an Elastic Defend advanced policy option for pruning capability arrays. ({kibana-pull}179766[#179766])
+* Adds {agent} activity flyout enhancements. ({kibana-pull}179161[#179161])
```

Comment:
```suggestion
* The "agent activity" flyout now includes several new features. ({kibana-pull}179161[#179161])
  ** A "review errors" button now appears above the agent listing table when new activity events are loaded that include errors. Clicking the button will open the activity flyout with these errors shown.
  ** Agent activity now supports pagination. Click the "show more" button at the bottom of the list to load additional activity events.
  ** Agent activity from a given date can now be loaded by clicking the "Go to date" button and selecting a date. 
```

Tried to mimic the sub-bullets that appear below here. Hope I got the formatting right!